pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -1253,7 +1253,39 @@ public void testFilteredAggregationWithNoValueMatchingAggregationFilterDefault()
   }
 
   @Test
-  public void testFilteredAggregationWithNoValueMatchingAggregationFilterWithOption()
+  public void testFilteredAggregationWithNoValueMatchingAggregationFilterDefault()
```

Comment:
This test fails with `ArrayIndexOutOfBoundsException` on `master` similar to the issue described in https://github.com/apache/pinot/issues/15121 and passes with the fix in this patch.